_data/alumni.yml

```diff
@@ -0,0 +1,17 @@
+members:
+- name: Michael Ashburner
```

Comment:
ideally we could also get departure dates for alumni, but I don't think that's feasible @nlharris